lib/main.dart

```diff
@@ -9,6 +11,9 @@ void main() async {
   await Firebase.initializeApp(
     options: DefaultFirebaseOptions.currentPlatform,
   );
+  SystemChrome.setPreferredOrientations([
+    DeviceOrientation.portraitUp,
```

Comment:
Great idea: this will prevent display problems and will limit the UI workload (only one orientation to take into account)